src/contents/all/components/knot/Knots/Knots.tsx

```diff
@@ -38,7 +41,7 @@ const Knots = (props: Props) => {
       {hasKnots &&
         props.knots.map((knot, index) =>
           <Knot
-            key={knot.id}
+            key={knot.id} //knot has no id?
```

Comment:
will remove comment

Reply:
dope! once you remove I'll approve and merge 🙌🏽